src/HaroohiePals.MarioKartToolbox/Gui/View/CourseEditor/CollisionCheatSheetModalView.cs

```diff
@@ -24,6 +24,7 @@ protected override void DrawContent()
             ImGui.BulletText("Color and Shadow can be put in any order or omitted");
             ImGui.BulletText("Light Colors: C, Col, Color + ID. Valid examples: Color2, C1, Col3");
             ImGui.BulletText("Shadow on map: S, Shd, Shadow");
+            ImGui.BulletText("Unused Flag: unused, uf, u");
```

Comment:
Please use the same token order and capitalization as the other examples:
```suggestion
            ImGui.BulletText("Unused Flag: U, Uf, Unused");
```